app.py

```diff
@@ -1,22 +1,92 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
 from flask import request
+import requests
+import os
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    items = session.get_items()
+    #items = session.get_items()
+    url = f"https://api.trello.com/1/boards/{os.getenv('BOARD_ID')}/cards"
+
+    
+
+    query = {
+    'key': os.getenv("KEY"),
+    'token': os.getenv("TOKEN")
+    }
```

Comment:
You could reduce some duplication in this file by having a get_auth_params function that returns this dictionary.

And then your various functions can use it and add to the dictionary if more params are needed.

Reply:
Please check the changes with commit ID - ad8f0c427b592c542b116bdb03e61ebd8ac26d80

Reply:
Doesn't look like you actually made changes relevant to this comment?